pkg/sanity/identity.go

```diff
@@ -79,6 +79,8 @@ var _ = Describe("Probe [Identity Service]", func() {
 		Expect(ok).To(BeTrue())
 		Expect(serverError.Code() == codes.FailedPrecondition ||
 			serverError.Code() == codes.OK).To(BeTrue())
+
+		Expect(res.GetReady().GetValue()).To(BeTrue())
```

Comment:
I'm not sure if we should expect this to be true always, but if it's not true everything might fail.

Reply:
Couple of things.

1. The value can be `true` or `false`. We should not make sure it is true, just make sure it is true or false.
2. We need to check first if the value is set since according to the spec, this is optional. So first we need to

```go
if req.GetRead() != nil {
    // Check the values 
}
```

Reply:
Thanks! That cleared my confusion :)